app/utils.py

```diff
@@ -127,12 +127,36 @@ def email_address_is_nhs(email_address):
     ))
 
 
-def update_dct_to_str(update_dct):
-    str = "\n"
+def update_dct_to_str(update_dct, lang):
+    if lang not in ['EN', 'FR']:
+        raise NotImplementedError
```

Comment:
Is there a cleaner way to do this on the API side? We don't have supported lang + translation files here.

Reply:
Correct. Right now translations typically happen by passing a key to the admin which handles the translating. The API itself doesn't do enough translation to have a system for it.

Unless we passed these values from the admin (which could be a good idea given the way our logic is laid out, but would be a more complicated fix). Otherwise, this would be the best way.